### ISSUE TITLE
Replacing SysUtils.ReportErrorToEventLog with SysUtils.ReportErrorMessageToEventLog

### ISSUE THREAD

**Issue by SoftAdviceJR** [M-Files/VAF.Extensions.Community#53]
In multiple places in the VAF.Extensions code, the method ReportErrorToEventLog is called where a large string is supplied as event source. This is prone to error as the maximum length for an event source string is 212 characters.

Most of these strings seem more like error descriptions and should be added messages instead of source identifiers.

I encountered the error at the following location:

```
   at System.Diagnostics.EventLog.CreateEventSource(EventSourceCreationData sourceData)
   at MFiles.VAF.Common.SysUtils.ReportToEventLog(String eventSource, String message, EventLogEntryType type)
   at MFiles.VAF.Common.SysUtils.ReportErrorToEventLog(String eventSource, Exception ex)
   at MFiles.VAF.Extensions.TaskQueueBackgroundOperation`1.CancelFutureExecutions(String remarks)
   at MFiles.VAF.Extensions.TaskQueueBackgroundOperation`1.RunOnSchedule(Schedule schedule, TaskDirective directive)
   at MFiles.VAF.Extensions.TaskQueueBackgroundOperationManager`1.StartScheduledBackgroundOperation[TDirective](String name, Schedule schedule, Action`2 method, TDirective directive)
```

**Comment by CraigHawker** [comment]
Hi Jeroen,

You are probably correct, but I am going to close this issue as the version which uses the logging framework does not use these methods (e.g. https://github.com/M-Files/VAF.Extensions.Community/blob/logging/MFiles.VAF.Extensions/TaskQueueBackgroundOperations/TaskQueueBackgroundOperationManager/CancelFutureExecutions.cs).

This will likely be released in the coming few days.  In the interim you can take a look at the (released) preview which contains these changes.

Regards,

Craig.

**Comment by CraigHawker** [comment]
I hate using the "wontfix" label as it comes across as ignoring the issue.  I'm not but, as the preview version already does not use this approach, and the preview version is likely to become the release version very soon, I think it's probably not something we can consider working on.

**Comment by CraigHawker** [comment]
As this change is relatively small, and the impact high, I have re-opened this.  I've done this for two reasons:

1. Moving to the new logging framework may not be something people are happy to do immediately, so may stay on 1.3 for a while.
2. It's likely we'll make at least one final release to the 1.3 branch.

**Comment by CraigHawker** [comment]
Fix is included in 1.3.1.5
